transport/conn.go

```diff
@@ -336,6 +365,9 @@ func (c *Conn) sendRequest(req frame.Request, compress, tracing bool) (frame.Res
 		ResponseHandler: h,
 	}
 
+	// Note: requestCh might be full after terminating writeLoop so some goroutines could hang here forever.
```

Comment:
Note is not needed just write what you have to say.
This is a feature, it could be put as a doc string of sendRequest indicating that it can block.

Reply:
I don't think that sendRequest possibly hanging forever is a good feature. Currently we are not guaranteed that closeRequest will be the last thing ever sent through requestCh, if more than requestChanSize requests will pile up after it those goroutines will get stuck forever.